pkg/events/core.go

```diff
@@ -12878,21 +12880,71 @@ var CoreEvents = map[ID]Definition{
 			{Type: "bool", Name: "load"},
 		},
 	},
+	ExecuteFinished: {
+		id:       ExecuteFinished,
+		id32Bit:  Sys32Undefined,
+		name:     "execute_finished",
+		version:  NewVersion(1, 0, 0),
+		sets:     []string{"proc"},
+		internal: true,
+		dependencies: Dependencies{
+			probes: []Probe{
+				{handle: probes.SyscallExit__Internal, required: true},
+			},
+			tailCalls: []TailCall{
+				{"sys_exit_init_tail", "syscall__execveat__execute_failed", []uint32{uint32(Execve), uint32(Execveat)}},
```

Comment:
Could be a leftover?

Kernels:

- `Linux hb0 6.6.10-1-MANJARO #1 SMP PREEMPT_DYNAMIC Fri Jan  5 17:38:36 UTC 2024 x86_64 GNU/Linux`
- `Linux ip-172-31-5-229 5.4.0-137-generic #154-Ubuntu SMP Thu Jan 5 17:03:22 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux`

`sudo ./dist/tracee -e process_execute_failed`

```json
{"level":"fatal","ts":1708628341.161495,"msg":"Tracee runner failed","error":"cmd.Runner.Run: error initializing Tracee: ebpf.(*Tracee).Init: ebpf.(*Tracee).initBPF: ebpf.(*Tracee).populateBPFMaps: failed to initialize tail call: ebpf.(*Tracee).initTailCall: could not get BPF program syscall__execveat__execute_failed: failed to find BPF program syscall__execveat__execute_failed: no such file or directory"}
```

Reply:
You are right, I changed the name but didn't update it here.

Reply:
It didn't pop up in the tests because I forgot to add the e2e to the `pr.yml` file...